test/libdnf/base/test_transaction.hpp

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
```

Comment:
See other comment about license boilerplates.

Reply:
TBD in separate issue.